pyop2/local_kernel.py

```diff
@@ -105,11 +105,6 @@ def __init__(self, code, name, accesses=None, *,
         self.requires_zeroed_output_arguments = requires_zeroed_output_arguments
         self.user_code = user_code
 
-    @property
-    @abc.abstractmethod
-    def dtypes(self):
```

Comment:
Do we actually want to remove dtypes from the abstract base class here?
I'm thinking specifically of a scenario where a new kernel subclasses the ABC